### PR TITLE
Update Configuration.php

### PR DESCRIPTION
**Pull request by parijke**
changed rootNode via constructor according to deprecation warning

### CODE REVIEW

File: src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -28,41 +28,42 @@ class Configuration implements ConfigurationInterface
 {
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder;
+        $treeBuilder = new TreeBuilder('surfnet_message_bird_api_client', 'string' );
```
Comment (MKodde) [inline src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php:31]: Comparing your change to some similar bundle Configuration files, the second param you passed ot the Tree builder is not required as it seems.

https://github.com/OpenConext/Stepup-saml-bundle/blob/develop/src/DependencyInjection/Configuration.php

File: src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -28,41 +28,42 @@ class Configuration implements ConfigurationInterface
...
+                            ->thenInvalid(
+                                "Authorization value '%s' should be in the format 'AccessKey your_access_key_here'."
+                            )
+                        ->end()
+                    ->isRequired()
+                ->end()
+            ->end();
 
-        return $treeBuilder;
+        return $rootNode;
```
Comment (MKodde) [inline src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php:67]: Here, you need to return `$treeBuilder` right?

**Review by MKodde — CHANGES_REQUESTED** [review]

File: src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php (hunk 2 above)
Comment (parijke) [reply src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php]: Adjusted accordingly

File: src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (parijke) [reply src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php]: removed send parameter

**Review by MKodde — APPROVED** [review]
:1st_place_medal:  looking good. I will shipit

**Pull request merged**